deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml

```diff
@@ -38,5 +37,7 @@ services:
     command:
       - bash
       - -c
-      - make integration_test_kafka CASE=kafka_messages
-    network_mode: "container:kafka"
+      - make integration_test_kafka CASE=clustered_index & tail -f /dev/null
+    network_mode: "service:kafka"
```

Comment:
just a question: i can't find the differences between container_network and service_network 
it's hard to find the doc about this option, but i find [this](https://github.com/docker/docker.github.io/issues/9725#issuecomment-761832882) 

> Otherwise it will ask to create the kafka container first.

Is this the reason for modifying to service network? btw why change test case to `clustered_index `

Reply:
> Is this the reason for modifying to service network? 

Yes. I'm trying in codespaces and using the container it asks to create the network first. Can you help to try it locally?

> btw why change test case to clustered_index

I will think of a better way to solve this problem.

Reply:
> CASE="clustered_index" docker-compose -f ./deployments/ticdc/docker-compose/docker-compose-kafka-integration.yml up
```
Step 5/30 : RUN ./download-integration-test-binaries.sh master
...
  0     0    0     0    0     0      0      0 --:--:--  0:01:50 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:02:07 --:--:--     0curl: (7) Failed connect to fileserver.pingcap.net:80; Connection timed out
The command '/bin/sh -c ./download-integration-test-binaries.sh master' returned a non-zero code: 7
```

seems this test only can running in IDC 😅

Reply:
Yes, it can only be used in an internal network environment.

Reply:
I don't know how to fix this yet, maybe it's to use tiup's mirror link. But that doesn't seem to guarantee that we'll get the latest nightly builds. Also some packages may not be in the mirror.